### PR TITLE
So anyways I did a few mappings Part 2!

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

File: mappings/net/minecraft/utill/shape/VoxelSet.mapping [hunk 1]
```diff
@@ -0,0 +1,12 @@
+CLASS net/minecraft/class_4086 net/minecraft/utill/shape/VoxelSet
```
Comment (arthurbambou) [inline mappings/net/minecraft/utill/shape/VoxelSet.mapping:1]: utill?

File: mappings/net/minecraft/utill/shape/SliceVoxelShape.mapping [hunk 2]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_4098 net/minecraft/utill/shape/SliceVoxelShape
```
Comment (arthurbambou) [inline mappings/net/minecraft/utill/shape/SliceVoxelShape.mapping:1]: utill?

**Review by arthurbambou — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/utill/shape/SliceVoxelShape.mapping (hunk 2 above)
Comment (u9g) [reply mappings/net/minecraft/utill/shape/SliceVoxelShape.mapping]: fixed in 31215e0

File: mappings/net/minecraft/utill/shape/VoxelSet.mapping (hunk 1 above)
Comment (u9g) [reply mappings/net/minecraft/utill/shape/VoxelSet.mapping]: fixed in 31215e0

**Review by arthurbambou — COMMENTED** [review]
this seems to have created conflicts again....

**Review by arthurbambou — APPROVED** [review]

**Comment by BleachDrinker420** [comment]
Why rename Box to BoundingBox?

**Comment by u9g** [comment]
> Why rename Box to BoundingBox?

I renamed box to boundingbox, since box was kind of vague, many things can be a box, a block is a box, a chest can be a box, etc. Should I change it back? It just made more sense to me this way.

**Comment by PhoenixVX** [comment]
The more appropriate name would be AxisAlignedBoundingBox, AABB, or AxisAlignedBB like in MCP. This bounding box usually represents collision boxes rather than a generic box. BoundingBox is okay, but it doesn't really tell you that this bounding box is aligned to an axis.

**Comment by BleachDrinker420** [comment]
ITS A BOX

**Comment by PhoenixVX** [comment]
A box doesn't tell you it's a hit box, a block box, an entity collision box... A form of Bounding Box is approriate.

**Comment by u9g** [comment]
> The more appropriate name would be AxisAlignedBoundingBox, AABB, or AxisAlignedBB like in MCP. This bounding box usually represents collision boxes rather than a generic box. BoundingBox is okay, but it doesn't really tell you that this bounding box is aligned to an axis.

Can I use their axisalignedboundingbox name? aren’t I supposed to not take from mcp?

**Comment by u9g** [comment]
AxisAlignedBox? ;p

**Comment by PhoenixVX** [comment]
Mojang also uses that name. As for the legality of using it, that's quite hard to say since it's quite easy to determine that's the functionality of it is appropriately represented by such name, i.e. coming up with the same name wouldn't be far-fetched.

**Comment by BleachDrinker420** [comment]
Its just a box with 8 corners, it can be used for anything it doesn't need any more info, and the main priority should anyway be to match the upstream fabric mappings

**Comment by PhoenixVX** [comment]
It used to be named BoundingBox in upstream, but it recently got changed to Box for some reason.

**Review by PseudoDistant — APPROVED** [review]
B O X

**Comment by u9g** [comment]
ok ok I din’t have that strong of an opinion, I’ll call it Box

**Review by BleachDrinker420 — APPROVED** [review]

**Review by arthurbambou — APPROVED** [review]

**Pull request merged**